get-started/deployment-options.md

```diff
@@ -1,17 +1,21 @@
 ---
+navigation_title: Deployment options
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/elasticsearch-intro-deploy.html
 products:
   - id: elasticsearch
 ---
 
-# Deployment options [elasticsearch-intro-deploy]
+# Deployment options
 
-Use this page for a quick overview of your options for deploying Elastic.
+You can run Elastic on virtually any infrastructure, allowing you to choose the model that best fits your operational needs. 
 
-:::{include} /deploy-manage/_snippets/deployment-options-overview.md
-:::
+These are the most common deployment types and their main features:
 
-:::{tip}
-Refer to [](/deploy-manage/index.md) for detailed information and guidance on choosing the right deployment for your needs.
-:::
+* **{{serverless-full}}**: This fully managed SaaS offering abstracts away all underlying infrastructure, automatically and seamlessly scaling resources to meet your workload demands. It's designed for operational simplicity, with usage-based pricing that allows you to focus on your data without managing clusters.  
```

Comment:
couple of downgrades from the original content:

* we had the [local development option](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/2795/deploy-manage/deploy/self-managed/local-development-installation-quickstart) displayed prominently so people who wanted to just explore our technology had a lightweight/quick way forward
* we categorized the options in terms of difficulty. I don't know the history of it but it was very intentional
* we were highlighting the ecloud trials, which was an ask from PM